configure.ac

```diff
@@ -233,7 +233,7 @@ AC_PATH_PROG(CMP, cmp)
 #
 
 PROSPECTIVE_CFLAGS="-Wall -Wextra -Wshadow -Werror -std=c99 -pedantic-errors"
-PROSPECTIVE_CXXFLAGS="-Wall -Wextra -Wshadow -Werror -std=gnu++98 -Wno-c++14-compat -fno-exceptions"
+PROSPECTIVE_CXXFLAGS="-Wall -Wextra -Wshadow -Werror -std=c++11 -pedantic-errors -fno-exceptions"
```

Comment:
We should define c++11 for Android build as well probably.

Reply:
And pedantic-errors as well.

Reply:
I updated `LOCAL_CPPFLAGS` in `Android.mk`.